<?php

namespace ContainerVatGb0R;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_MR3FjBXService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.mR3FjBX' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.mR3FjBX'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'entityManager' => ['services', 'doctrine.orm.default_entity_manager', 'getDoctrine_Orm_DefaultEntityManagerService', false],
            'user' => ['privates', '.errored..service_locator.mR3FjBX.App\\Entity\\User', NULL, 'Cannot autowire service ".service_locator.mR3FjBX": it references class "App\\Entity\\User" but no such service exists.'],
            'userPasswordHasher' => ['privates', 'security.user_password_hasher', 'getSecurity_UserPasswordHasherService', true],
        ], [
            'entityManager' => '?',
            'user' => 'App\\Entity\\User',
            'userPasswordHasher' => '?',
        ]);
    }
}
